commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/service/persistence/impl/CommerceInventoryWarehouseItemFinderImpl.java

```diff
@@ -41,6 +46,14 @@
 		CommerceInventoryWarehouseItemFinder.class.getName() +
 			".countStockQuantityByC_G_S";
 
+	public static final String COUNT_UPDATED_ITEMS_BY_M =
```

Comment:
I think this should be `..._BY_C_M` to match the exact finder you're using, rather than simplifying it to just `..._BY_M`.